composer.json

```
@@ -18,36 +18,49 @@
     }
   ],
   "support": {
-    "issues": "https://github.com/ifox/twill/issues/",
+    "issues": "https://github.com/area17/twill/issues/",
     "docs": "https://twill.io/docs",
     "chat": "https://discord.link/twill"
   },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "git@github.com:area17/laravel-translatable.git"
+    },
+    {
+      "type": "vcs",
+      "url": "git@github.com:ifox/tags.git"
+    },
+    {
+      "type": "vcs",
+      "url": "git@github.com:area17/laravel-nestedset.git"
+    }
+  ],
   "require": {
     "php": "^7.1|^8.0",
     "ext-json": "*",
     "ext-pdo": "*",
-    "astrotomic/laravel-translatable": "^11.5",
-    "cartalyst/tags": "^6.0|^7.0|^8.0|^9.0|^10.0|^11.0",
+    "astrotomic/laravel-translatable": "^11.10",
```

```suggestion
    "astrotomic/laravel-translatable": "^11.5,
```

Since we're merging Laravel 9 support on 2.x, we need to keep 11.5 as the lowest version to stay compatible with Laravel below 8.